.github/workflows/ci-bats-unix.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
       - name: Install Maven
         working-directory: ./.ci_bin
         run: |
-          curl -LO https://dlcdn.apache.org/maven/maven-3/3.9.2/binaries/apache-maven-3.9.2-bin.tar.gz
+          curl -LO https://dlcdn.apache.org/maven/maven-3/3.9.3/binaries/apache-maven-3.9.3-bin.tar.gz
           tar -xf apache-maven-3.9.2-bin.tar.gz
```

Comment:
Is there a reason the name of the tar.gz wasn't updated too?

Reply:
nope, just missed it